### ISSUE TITLE
Sync issues

### ISSUE THREAD

**Issue by cmbirk** [opengovfoundation/issuetotask#3]
# GH -> Teamwork
- [ ]  Pull all current issues from GH into Teamwork.
- [ ]  Set up webhooks for issue closure in GH.
- [ ] Create API connector for GH issue closure
  - Make sure to store GH issue ID
- [ ]  Test issue closure
# Teamwork -> GH
- [ ] Set up webhooks for task completion
- [ ] Create API connector for GH issue closure

**Comment by cmbirk** [comment]
Waiting on feedback from the TW team on their Webhook API endpoints

**Comment by cmbirk** [comment]
Closing this as out of scope.  We determined the sync would be only GH->TW for now.
